extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -188,6 +202,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    this.lifecycleReporter_.sendPing('layoutCallback');
```

Comment:
I feel sending pings at these points are not very helpful data. For example render-start would be a better place to do it.
Actually, I will suggest to split off the 3p changes from this PR.